controllers/update_status.go

```diff
@@ -175,23 +175,6 @@ func (s UpdateStatus) Reconcile(r *FoundationDBClusterReconciler, context ctx.Co
 		status.ProcessGroups = append(status.ProcessGroups, fdbtypes.NewProcessGroupStatus(processGroupID, internal.ProcessClassFromLabels(service.Labels), nil))
 	}
 
-	// Ensure that anything the user has explicitly chosen to remove is marked
-	// for removal.
-	for _, processGroupID := range cluster.Spec.InstancesToRemove {
```

Comment:
This will already be checked in validate instances with `InstanceIsBeingRemoved`